File: python/src/Behavioral/Interpreter/DvdActorTitleExpression.py
#//DvdActorTitleExpression - Two Of Four Terminal Expressions

define ['Behavioral/Interpreter/DvdAbstractExpression'
],(
DvdAbstractExpression
) ->

	class DvdActorTitleExpression extends DvdAbstractExpression
		constructor : (@title) ->
		interpret : (ctx) -> ctx.getActorsForTitle(@title).join ', '

File: ruby/src/Behavioral/Interpreter/DvdActorTitleExpression.rb
#//DvdActorTitleExpression - Two Of Four Terminal Expressions

require "Behavioral/Interpreter/DvdAbstractExpression"

class DvdActorTitleExpression < DvdAbstractExpression
	def initialize(title)
		@title = title
	end

	def interpret(ctx)
 		return ctx.getActorsForTitle(@title).sort.join(', ');
	end
end
